previews/primer/box_preview.rb

```diff
@@ -11,16 +11,19 @@ def playground(content: "Playground")
     end
 
     # @label Default
+    # @snapshot
```

Comment:
Not sure these are useful 
![image](https://github.com/primer/view_components/assets/54012/9e720bb2-8b5f-42c6-a56e-6d05e33a810c)